app/scss/adminPage.scss

```diff
@@ -199,3 +202,16 @@ input[type="number"] {
   }
 }
 
+.drop-down-info-panel {
+  display: flex;
+  flex-flow: row nowrap;
+  //background-color: purple;
+}
+
+.more-info-data {
+  width: 100%;
+}
+
+.unShow {
+    display: none;
+}
```

Comment:
this is the same as the bootstrap class `hide` which has been used elsewhere - delete this and replace usages with `hide` and make sure nothing breaks

Reply:
this done thanks to the PO guidance